Plugins/AGSSpriteFont/AGSSpriteFont/SpriteFontRenderer.h

```diff
@@ -7,7 +7,7 @@ class SpriteFontRenderer :
 {
 public:
 	SpriteFontRenderer(IAGSEngine *engine);
-	~SpriteFontRenderer(void);
+	virtual ~SpriteFontRenderer(void);
```

Comment:
Note that in the IAGSFontRenderer interface destructors are not virtual, this was kept so deliberately, because these are not supposed to be deleted by IAGSFontRenderer pointer.

Reply:
Even if it is protected, it still should be virtual to avoid invoking undefined behaviour.
If the destructor is never called this way, what should be the correct cleanup procedure?

Reply:
I reverted the change, even though I think it makes sense to have it.